webapp/src/components/open-api-plugins/PluginGallery.tsx

```diff
@@ -93,7 +89,12 @@ export const PluginGallery: React.FC = () => {
                         setHostedPlugins((hostedPlugins) => [...hostedPlugins, newHostedPlugin]);
                     })
                     .catch((error: Error) => {
-                        dispatch(addAlert({ message: error.message, type: AlertType.Error }));
+                        dispatch(
+                            addAlert({
+                                message: `Failed fetch hosted plugins: ${error.message}`,
```

Comment:
Failed TO fetch...

Also, this is done for each plugin, right?  So it should be Failed to fetch hosted plugin (no s at then) and the error message would be better if it include info as to _which_ plugin failed.

Reply:
Good catch!